README.md

```diff
@@ -56,9 +56,9 @@ And then follow on-screen prompt:
    
 4. Enter query. This can be obtained from Copernicus Open Hub `Request Done: ( ... )`. Just copy that whole string.
 
-5. Enter Polling Interval. Entering nothing will use default.
+5. Enter Polling Interval. Entering nothing will use default of one poll every 30 mins.
 
-6. Enter offline product download retries. Entering nothing will use default.
+6. Enter offline product download retries. Entering nothing will use default of 24 retries. That is half a day's grace for the product to be online before moving on to the next product. 
```

Comment:
Just to make sure that I understand this correctly, it takes half a day because polling interval is 30 mins and offline product retries is 24 right?

Reply:
Yup! should i rephrase?

Reply:
```suggestion
6. Enter offline product download retries. Entering nothing will use default of 24 retries. That is half a day's grace (for default polling interval of 30 mins) for the product to be online before moving on to the next product. 
```